layouts/partials/card.html

```diff
@@ -19,34 +19,32 @@
         <div class="hover-card-video">
 					{{ partial "gif.html" (dict "webm_src" .Params.webmSrc "mp4_src" .Params.mp4Src "alt" .Params.videoAlt "maxWidth" "100%" "class" "" ) }}
         </div>
-        {{ end }}
-        {{ if .Params.image }}
-          <div class="hover-card-img">
-            {{ $img := resources.GetMatch .Params.image }}
-            {{if $img }}
-              {{ partial "imgproc.html" (dict "src" (.Params.image) "resize" "400x" "alt" (.Params.imageAlt) "style" "") }}
-            {{ else }}
-              {{ errorf "Couldn't get image %s" .Params.image }}
-            {{ end }}
-          </div>
-        {{ end }}
-				{{ if .Params.icon }}
-				<div class="hover-card-icon">
-					{{ partial "imgproc.html" (dict "src" (.Params.icon) "resize" "400x" "alt" (.Params.linkTitle) "style" "") }}
-				</div>
-				{{ end }}
-        <div {{ if .Params.image }}class="small-hover-card-div"{{ end }}{{ if .Params.webmSrc }}class="small-hover-card-div"{{ end }}>
-          {{ if $customTitle }}
-            <div>{{ $customTitle }}</div>
+      {{ else if .Params.images }}
+        <div class="hover-card-img">
+          {{ $img := resources.GetMatch (index .Params.images 0) }}
+          {{ if $img }}
+            {{ partial "imgproc.html" (dict "src" (index .Params.images 0) "resize" "400x" "alt" (.Params.imageAlt) "style" "") }}
```

Comment:
I believe this is all that needs to change. The problem is with pages where there is an images field with an image in it but no alt text. Alternatively you could add an alt text for each such page.

```suggestion
            {{ if .Params.imageAlt }}
              {{ partial "imgproc.html" (dict "src" (index .Params.images 0) "resize" "400x" "alt" (.Params.imageAlt) "style" "") }}
            {{ else }}
              {{ partial "imgproc.html" (dict "src" (index .Params.images 0) "resize" "400x" "alt" (.Params.linkTitle) "style" "") }}
              {{ warnf "using linktitle for %s" (index .Params.images 0)}}
            {{ end }}
            {{ else }}
```